pkg/handlers/update_checker_spec.go

```diff
@@ -85,9 +179,11 @@ func (h *Handler) ConfigureAutomaticUpdates(w http.ResponseWriter, r *http.Reque
 		}
 	}
 
-	if err := store.GetStore().SetUpdateCheckerSpec(foundApp.ID, cronSpec); err != nil {
+	cm.Data[fmt.Sprintf("%s-spec", foundApp.ID)] = cronSpec
```

Comment:
key name should be less generic since this is config map is not a specifically for update checks.  How about `"update-schedule-%s"`

Reply:
done